### PR TITLE
Story 4

### PR DESCRIPTION
**Pull request by Phildallen**

### CODE REVIEW

File: src/Database/SongDao.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ public function fetchSongFromSongId(int $songId): Song
         $query->execute($value);
         $song = $query->fetch();
 
-        return new Song($song['id'], $song['song_name'], $song['length'], $song['album_id']);
+        return new Song($song['id'], $song['song_name'], $song['length'], $song['album_id'], $song['song_count']);
```
Comment (nealioa) [inline src/Database/SongDao.php]: Change `song_count` to `play_count`.

File: src/Database/SongDao.php [hunk 2]
```diff
@@ -43,6 +43,45 @@ public function fetchAllSongsFromAlbumId(int $albumId): array
         return $songs;
     }
 
+
```
Comment (nealioa) [inline src/Database/SongDao.php]: Two blank lines! We only need one. :-)

File: src/Database/SongDao.php [hunk 3]
```diff
@@ -43,6 +43,45 @@ public function fetchAllSongsFromAlbumId(int $albumId): array
...
+            .'SET `play_count` = `play_count` + 1 '
+            .'WHERE `id` = :id;';
+        $value = [':id' => $id];
+
+        $stmt = $this->db->getPdo()->prepare($sql);
+
+        $success = $stmt->execute($value);
+        return $success;
+    }
+
```
Comment (nealioa) [inline src/Database/SongDao.php:82]: Two blank lines! We only need one. :-)

**Review by nealioa — CHANGES_REQUESTED** [review]

File: src/Database/SongDao.php (hunk 1 above)
Comment (Phildallen) [reply src/Database/SongDao.php]: Done

File: src/Database/SongDao.php (hunk 2 above)
Comment (Phildallen) [reply src/Database/SongDao.php]: Done

File: src/Database/SongDao.php (hunk 3 above)
Comment (Phildallen) [reply src/Database/SongDao.php:82]: done

**Review by nealioa — APPROVED** [review]

**Pull request merged**